src/app/app.component.ts

```diff
@@ -14,12 +15,15 @@ export class AppComponent implements OnInit {
 
     constructor(
         private _router: Router,
-        private _titleService: Title
+        private _titleService: Title,
+        private _datadogRumService: DatadogRumService
     ) {
 
         // set the page title
         this._titleService.setTitle('DaSCH Service Platform');
 
+        // init datadog RUM
+        this._datadogRumService.initializeRum();
```

Comment:
Why is the `datadogRumService` initialized here and not at time of construction?

Reply:
changed in a48ea3d